<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionsDialog</class>
 <widget class="QDialog" name="optionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>378</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Incident Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="datumField">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>NAD27 CONUS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WGS84</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Coordinate Format</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="timeoutField">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Datum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="formatField">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>UTM 5x5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UTM 7x7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D.d°</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D° M.m'</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D° M' S.s&quot;</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="timeoutLabel">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Timeout</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="incidentField">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>New Incident</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="secondWorkingDirCheckBox">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="whatsThis">
      <string extracomment="Uncheck if local network is slow or intermittent"/>
     </property>
     <property name="text">
      <string>Use Second Working Directory</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>optionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>optionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>secondWorkingDirCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>optionsDialog</receiver>
   <slot>secondWorkingDirCB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>secondWorkingDirCB()</slot>
 </slots>
</ui>
